scripts/Dump_MAT_APP_BLANK_V6.sql

```diff
@@ -1662,17 +2440,45 @@ CREATE TABLE `USER_PASSWORD` (
 
 LOCK TABLES `USER_PASSWORD` WRITE;
 /*!40000 ALTER TABLE `USER_PASSWORD` DISABLE KEYS */;
-INSERT INTO `USER_PASSWORD` VALUES ('1','Admin',0,0,'8a06ddf2a3da6e7d558c91951fb48d3f5787906904724b25d23ff161f92b1e70','d0dccfba-9178-4466-9a1a-981023b721a9',1,sysdate(),NULL,0);
+INSERT INTO `USER_PASSWORD` VALUES ('1','Admin',0,0,'8a06ddf2a3da6e7d558c91951fb48d3f5787906904724b25d23ff161f92b1e70','d0dccfba-9178-4466-9a1a-981023b721a9',1,'2019-10-18',NULL,0);
```

Comment:
I'm not sure but hardcoded CREATE_DATE might cause problems for **temporary password expiration** because created date + 5 days(temp password validity) will always be in past if you see this few days later.

Reply:
I have gone in and updated this to sysDate().

Reply:
thanks nicole